zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -573,3 +608,122 @@ def detect_sriov_capability(cls, pci_device_to, gpu_info_map=None):
+    def detect_tensorfusion_capability(cls, pci_device_to):
+        """
+        Detect NVIDIA TensorFusion (GPU virtualization) capability.
+
+        Requirements:
+        - NVIDIA driver version >= 570.x
+
+        Returns tuple: (is_supported, capability_info)
+        """
+        addr = pci_device_to.pciDeviceAddress
+
+        r, o, e = bash_roe('nvidia-smi --query-gpu=pci.bus_id,driver_version --format=csv,noheader -i %s' % addr)
```

Ruff (0.15.9)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)